aten/src/ATen/native/mps/operations/UnaryOps.mm

```diff
@@ -453,4 +457,32 @@ static void cumulative_op_impl(const Tensor& self,
   return cumulative_op_impl(self, dim, dtype, result, MPSCumulativeOpType::CUMPROD, "cumprod_out_mps");
 }
 
+TORCH_IMPL_FUNC(sgn_out_mps)(const Tensor& self, const Tensor& output) {
+  if (!self.is_complex()) {
+    Tensor output_copy = output.alias();
+    at::sign_out(output_copy, self);
+    output.copy_(output_copy);
+    return;
```

Comment:
```suggestion
    return sign_out_mps(self, output);
```

Reply:
This causes an error, I think because sign_out_mps is declared with the TORCH_IMPL_FUNC macro. 

I was also trying to get rid of the alias() and subsequent copy_() call. Do you see another way to do it? The obstacle as I understand it is that output is const and the available sign_out functions take non-const arguments for output.

Reply:
This should work, but I'm not sure if it's preferable:

`output.copy_(at::sign(self));
return;`

Reply:
@igm503, no this would be much slower, as it would incur extraneous copy.
Anyway, let's land as is and I'll submit a follow up PR to fix this.